core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -726,6 +726,12 @@ public enum CoreError implements ScalarDbError {
       "This TIMESTAMPTZ column value precision cannot be shorter than one millisecond. Value: %s",
       "",
       ""),
+  JDBC_IMPORT_DATA_TYPE_OVERRIDE_NOT_SUPPORTED(
+      Category.USER_ERROR,
+      "0158",
+      "The underlying storage data type %s is not supported as ScalarDB %s data type: %s",
+      "",
+      ""),
```

Comment:
@josh-wong 
Do you mind checking the English of this error message?

Reply:
The following suggestion is just a minor change.

```suggestion
  JDBC_IMPORT_DATA_TYPE_OVERRIDE_NOT_SUPPORTED(
      Category.USER_ERROR,
      "0158",
      "The underlying-storage data type %s is not supported as the ScalarDB %s data type: %s",
      "",
      ""),
```